src/main/java/racingcar/domain/Car.kt

```diff
@@ -0,0 +1,16 @@
+package racingcar.domain
+
+import racingcar.strategy.move.MoveStrategy
+
+data class Car(
+    val name: Name,
+    val position: Position = Position()
```

Comment:
원시값 포장 👍💯 

조금 더 첨언해보자면, Position이 0인 경우는 자주 쓰이게 될 것 같은데,
position의 default parameter를 아래와 같이 ZERO와 같은 상수를 하나 만들어서 사용해보면 어떨까요? 

```suggestion
    val name: Name,
    val position: Position = Position.ZERO
```

Reply:
오 넵넵 default parameter 값을 좀 더 명확하게 표현해줄 수 있겠네요 👍